lib/jekyll/filters.rb

```diff
@@ -323,6 +325,23 @@ def sort_input(input, property, order)
         .map!(&:last)
     end
 
+    # `where` filter helper
+    def compare_property_vs_target(property, target)
+      case target
+      when NilClass
+        # won't catch `""`
+        return true if Array(property).empty?
```

Comment:
Why are we converting to an array here vs. just calling `.nil?` or something similar?

Reply:
Ah! You're right. This can be simplified to remove the overlap with Liquid's `empty` literal. :+1: